### PR TITLE
Add Symfony 6 support

### PR DESCRIPTION
**Pull request by coldic3**
To be merged on Sylius 1.12 release :woman_dancing:

In addition, https://github.com/Sylius/InvoicingPlugin/pull/295 and https://github.com/Sylius/InvoicingPlugin/pull/294 should be merged before this one then the rebase will be needed.

### CODE REVIEW

File: tests/Application/config/packages/webpac_encore.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+webpack_encore:
```
[the remark this answers is not in the document]
Comment (coldic3) [reply tests/Application/config/packages/webpac_encore.yaml]: fixed 🙂

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -20,21 +20,25 @@ jobs:
             fail-fast: false
             matrix:
                 php: ["8.0", "8.1"]
-                symfony: ["^5.4"]
+                symfony: ["^5.4", "^6.0"]
                 sylius: ["~1.11.0", "~1.12.0"]
```
Comment (TheMilek) [inline .github/workflows/build.yml:24]:
```suggestion
                sylius: ["~1.11.10", "~1.12.0"]
```
We should probably change this as it requires 1.11.0 with PHP [8.1](https://github.com/Sylius/InvoicingPlugin/actions/runs/3392719526/jobs/5639232073#step:13:160) and 1.11.10 with [8.0](https://github.com/Sylius/InvoicingPlugin/actions/runs/3392719526/jobs/5639231592#step:13:161) 🤔

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -20,21 +20,25 @@ jobs:
...
-                        node: "14.x"
+                        symfony: "^6.0"
+                        sylius: "~1.12.0"
+                        node: "16.x"
                         mysql: "8.0"
                         pdf: false
 
+                exclude:
+                    -   symfony: "^6.0"
+                        sylius: "~1.11.0"
```
Comment (TheMilek) [inline .github/workflows/build.yml:40]:
```suggestion
                        sylius: "~1.11.10"
```

**Review by NoResponseMate — APPROVED** [review]

**Review by Rafikooo — APPROVED** [review]

**Review by TheMilek — APPROVED** [review]

**Review by jakubtobiasz — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Kevin! :tada:

**Pull request merged**